spartan_parallel/Cargo.toml

```diff
@@ -25,9 +25,11 @@ serde = { version = "1", features = ["derive"], default-features = false }
 bincode = { version = "1", default-features = false }
 subtle = { version = "2", features = ["i128"], default-features = false }
 zeroize = { version = "1", default-features = false, features = ["alloc"] }
-itertools = { version = "0.13", default-features = false }
+itertools = { version = "0.10", default-features = false }
 colored = { version = "2", default-features = false, optional = true }
 flate2 = { version = "1" }
+ceno-goldilocks = { git = "https://github.com/scroll-tech/ceno-Goldilocks.git", branch = "fix/crate-ready" }
```

Comment:
You should be able to just contribute your fix the main branch of that repository?  Let me have a look.

Reply:
I generally don't push to `main` without PR review, so here's one created: https://github.com/scroll-tech/ceno-Goldilocks/pull/2

Reply:
Yes, I meant create a PR etc. :)  Thanks!

Reply:
Why do you prefer to import `goldilocks` crate as `ceno-goldilocks`?

Reply:
I tried to import the main branch again, but I kept getting this error:
```
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /Users/raygao/.cargo/git/checkouts/ceno-goldilocks-e1f7db6a8c06e8fe/29a15d1/src/lib.rs:4:1
  |
4 | #![feature(generic_const_exprs)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0554`.
error: could not compile `goldilocks` (lib) due to 1 previous error
```

I thought it's because of toolchain versioning issue but after merging in the toolchain fix this error still occurs.

Reply:
Fixed. Now importing main branch of `ceno-goldilocks` as `goldilocks`.